components/popover/README.md

```diff
@@ -51,6 +51,16 @@ render(
 
 The component accepts the following props. Props not included in this set will be applied to the element wrapping Popover content.
 
+### focusFirstElement
+
+Focus the *first tabblable element* inside the `Popover` if `focusOnMount` is enabled. This is the default behaviour; set this flag to `false` to focus the container, requiring the user to manually tab to the first element inside the `Popover`.
+
+Has no effect if `focusOnMount` is set to `false`.
```

Comment:
Just an idea: maybe we could deprecate `focusOnMount` and use an enum-like property with three states instead?

```jsx
<Popover autofocus="firstElement" /> // focuses the first tabbable element on mount
<Popover autofocus="container" /> // focuses the container on mount
<Popover autofocus="none" /> // disables focusing on mount
```

Reply:
I’m cool with that actually. What’s our way to throw errors or warn developers when they invariably type something not in the enum list? Mind if I just throw an error?

Reply:
Actually, I wonder about using `autofocus` because there's already an `<input>` prop `autofocus` in HTML.

🚲🏠!

Reply:
Maybe leave the name as is then, i.e. `focusOnMount="firstElement"`.